site/src/pages/components/interest-invokers.explainer.mdx

```diff
@@ -0,0 +1,199 @@
+---
+menu: Active Proposals
+name: Interest Invokers (Explainer)
+layout: ../../layouts/ComponentLayout.astro
+---
+
+- Authors: [Keith Cirkel](https://github.com/keithamus)
+
+{/* START doctoc generated TOC please keep comment here to allow auto update */}
+{/* DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE */}
+
+# Interest Invokers
+
+## Pitch in Code
+
+```html
+<button interesttarget="my-tooltip">Hover/Focus to show the tooltip</button>
+
+<span popover=hint id="my-toolip">This is the tooltip</span>
+```
+
+## Introduction
+
+Following [invokers](../invokers.explainer), adding an `interesttarget`
+attribute to interactive elements: starting with `<button>`, `<input
+type="button">`/`<input type="reset">`, and `<a>` (perhaps expanding to
+`<area>`, `<input>`, `<textarea>`, `<select>`, `<summary>` or maybe more, see
+[#14](https://github.com/openui/open-ui/issues/908)). This would allow
+disclosure of high fidelity tooltips in a more accessible and declarative way.
+Elements with `interesttarget` will - when hovered, long pressed, or focussed -
```

Comment:
Maybe call out the long press being overloaded and more thought needed on mobile for example chrome android shows a context menu

Reply:
Agreed. I've been thinking that the exact methods for showing interest **need** to be UA-determined anyway. We can't spec eye-tracking. So for mobile, perhaps long-press followed by selection of something in UA-provided context menu?